storage/experimental/experimental.go

```diff
@@ -25,9 +25,27 @@ import (
 	"time"
 
 	"cloud.google.com/go/storage/internal"
+	"go.opentelemetry.io/otel/sdk/metric"
 	"google.golang.org/api/option"
 )
 
+// WithMetricInterval provides a [ClientOption] that may be passed to [storage.NewGrpcClient].
+// It sets how often to emit metrics when using NewPeriodicReader and must be
+// greater than 1 minute.
```

Comment:
Why is this the minimum? Not sure why this requires validation, users might have their own systems with higher rate limits.

Reply:
Good point; could pass along a value if it's non-zero.